webapp/platform/components/src/generic_modal/generic_modal.tsx

```diff
@@ -278,12 +294,14 @@ export const GenericModal: React.FC<Props> = ({
             enforceFocus={enforceFocus}
             onHide={onHideCallback}
             onExited={onExited}
-            backdrop={backdrop}
+            backdrop={shouldRenderBackdrop ? backdrop : false}
+            backdropStyle={isStacked ? {zIndex: 1051} : undefined}
```

Comment:
I think @M-ZubairAhmed was trying to get the different zIndexes standardized, but not sure if he finally did it.